src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -347,8 +353,18 @@ public void snapshotCreateBQDataset(BigQueryProject bigQueryProject, Snapshot sn
         if (bigQueryProject.datasetExists(snapshotName)) {
             bigQueryProject.deleteDataset(snapshotName);
         }
+
+        // TODO: When we support multiple datasets per snapshot, this will need to be reworked
```

Comment:
There are many places where we have the form `snapshot.getSnapshotSources().get(0)` and each place now has this comment. It would be better to put this hack in a method in `Snapshot` and put the comment there, so we don't need to comment it everywhere.